src/lib/KafkaTypes.ts

```diff
@@ -10,3 +19,5 @@ export type KafkaProducerTopicConfig<T> = Omit<ProducerRecord, "messages"> & {
 
 export type KafkaProducerCompactConfig<T> = ValidableKafkaProducerConfig &
   KafkaProducerTopicConfig<T>;
+
+export type KafkaConsumerPayload = EachMessagePayload | EachBatchPayload;
```

Comment:
Do we need `EachMessagePayload ` ?

Reply:
This variable is not used anymore, I'll remove it.